docker/install.sh

```diff
@@ -6,7 +6,7 @@ SAM_DIR=$1
 cd $SAM_DIR
 
 # Tests are run in jenkins which has a custom opendj instance just for testing
-export SBT_OPTS="-Xms2g -Xmx2g -Ddirectory.url=$DIRECTORY_URL -Ddirectory.password=$DIRECTORY_PASSWORD"
+export SBT_OPTS="-batch -Xms2g -Xmx2g -Ddirectory.url=$DIRECTORY_URL -Ddirectory.password=$DIRECTORY_PASSWORD"
```

Comment:
Seems like these may be getting used as java opts, and java doesn't know what `-batch` is. https://fc-jenkins.dsp-techops.broadinstitute.org/job/sam-build/1788/console

Reply:
awww. the one time i tried to be clever. thanks